### PR TITLE
added Allocation Chart

### PR DESCRIPTION
**Pull request by yuliiagorgozka**

### CODE REVIEW

**Comment by yuliiagorgozka** [comment]
https://github.com/polywrap/polyfolio/issues/52

File: src/common/components/Button1D/Button1D.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import MenuArrow from 'common/components/MenuArrow/MenuArrow';
+import styles from './Button1D.module.scss';
+import classNames from 'classnames';
+
+function Button1D({fontSize, className}) {
+  return (
+    <button style={{fontSize}} className={classNames(styles.btn, className)}>
+      1D <MenuArrow className={styles.arrow} filled startPosition={'up'} />
```
Comment (DmitriyIvchenko) [inline src/common/components/Button1D/Button1D.tsx:9]: 1D? what does it mean?

**Review by DmitriyIvchenko — CHANGES_REQUESTED** [review]

File: src/common/components/Button1D/Button1D.tsx (hunk 1 above)
Comment (yuliiagorgozka) [reply src/common/components/Button1D/Button1D.tsx:9]: Repeated 1D button in all tables. I can just call it something else.

Comment (DmitriyIvchenko) [reply src/common/components/Button1D/Button1D.tsx:9]: but it is not button 1D it is table date range selector

Comment (DmitriyIvchenko) [reply src/common/components/Button1D/Button1D.tsx:9]: also 1 **day / week / month / year** should be moved to translations

Comment (yuliiagorgozka) [reply src/common/components/Button1D/Button1D.tsx:9]: Now I will throw a commit, and rename

**Review by DmitriyIvchenko — APPROVED** [review]

**Pull request merged**